cmd/docker-driver/Dockerfile

```diff
@@ -1,5 +1,23 @@
-FROM       alpine:3.9
-RUN        apk add --update --no-cache ca-certificates
-COPY       docker-driver /bin/docker-driver
+# Directories in this file are referenced from the root of the project not this folder
+# This file is intented to be called from the root like so:
+# docker build -t grafana/loki -f cmd/loki/Dockerfile .
+
+# TODO: really add cross-platform support
```

Comment:
imo I'd rather see the TODO left in here with all the other cross-platform comments taken out; when we get around to it we can refer to other Dockerfiles for how cross-platform support is added